src/qibo/backends/clifford.py

```diff
@@ -589,6 +85,29 @@ def zero_state(self, nqubits: int):
         symplectic_matrix[nqubits:-1, nqubits : 2 * nqubits] = self.np.copy(I)
         return symplectic_matrix
 
+    def clifford_pre_execution_reshape(self, state):
+        """Reshape the symplectic matrix to the shape needed by the engine before circuit execution.
+
+        Args:
+            state (ndarray): The input state.
+
+        Returns:
+            (ndarray): The reshaped state.
+        """
+        return self.engine.clifford_pre_execution_reshape(state)
+
+    def clifford_post_execution_reshape(self, state, nqubits):
+        """Reshape the symplectic matrix to the shape needed by the engine after circuit execution.
+
+        Args:
+            state (ndarray): The input state.
+            nqubits (int): Number of qubits.
+
+        Returns:
+            (ndarray): The reshaped state.
+        """
+        return self.engine.clifford_post_execution_reshape(state, nqubits)
+
```

Comment:
Do these two functions need to be user-facing?

Reply:
no you'e right, should I call them `_clifford_pre_execution_reshape` and `_clifford_post_execution_reshape`

Reply:
I think so